sdcm/cluster.py

```diff
@@ -2086,7 +2086,7 @@ def monitoring_version(self):
     def is_enterprise(self):
         return self.targets["db_cluster"].nodes[0].is_enterprise
 
-    def node_setup(self, node, **kwargs):
+    def node_setup(self, node, timeout=30, **kwargs):
```

Comment:
why to change default timeout for monitor node? it's not relative to adding a new cluster node

Reply:
I set it to 30 because of the timeout was transferred just from _add_and_init_new_cluster_node function and it was 30. In other usages of this function the timeout is not transferred.

Reply:
_add_and_init_new_cluster_node called from nemesis.py and works for CLUSTER nodes only, it's not relative to MONITOR node

Reply:
It failed on the monitor/node_setup because of unknown timeout parameter. So I added it there also. Retrying to run without timeout again.